src/components/button/inverse/index.njk

```diff
@@ -0,0 +1,12 @@
+---
+title: Inverse button
+layout: layout-example.njk
+stylesheets:
+  - ../../../stylesheets/example-inverse.css
+---
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{{ govukButton({
+  text: "Create an account",
+  classes: "govuk-button--inverse"
+}) }}
```

Comment:
We don't run any [`.editorconfig`](https://github.com/alphagov/govuk-design-system/blob/main/.editorconfig) checks but got a missing new line here